<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeometryConfigDialog</class>
 <widget class="QDialog" name="GeometryConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>410</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="geometryConfigLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Geometry Configuration</string>
   </property>
  </widget>
  <widget class="Line" name="headerSeparator">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>441</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="footerSeparator">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>441</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="ballGroupBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>280</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Ball:</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="ballFormLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="ballRadiusLabel">
       <property name="text">
        <string>Radius:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="ballRadiusInput"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="robotGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>211</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Robot:</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>171</width>
      <height>64</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="robotFormLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="robotRadiusLabel">
       <property name="text">
        <string>Radius:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="robotRadiusInput"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="chordDistLabel">
       <property name="text">
        <string>Chord distance:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="chordDistInput"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="fieldGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>211</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Field:</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>171</width>
      <height>98</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="fieldFormLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="fieldLengthLabel">
       <property name="text">
        <string>Length:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="fieldWidthLabel">
       <property name="text">
        <string>Width:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="centralRadiusLabel">
       <property name="text">
        <string>Central radius:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="fieldLengthInput"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="fieldWidthInput"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="centralRadiusInput"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="areaGroupBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>211</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Area:</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>171</width>
      <height>132</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="areaFormLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="areaLineLabel">
       <property name="text">
        <string>Line:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="areaRadiusLabel">
       <property name="text">
        <string>Radius:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="goalWidthLabel">
       <property name="text">
        <string>Goal width:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="areaLineInput"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="areaRadiusInput"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="goalWidthInput"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="goalDepthLabel">
       <property name="text">
        <string>Goal depth:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="goalDepthInput"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="milimitersLabel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>280</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>32</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>mm</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GeometryConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GeometryConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
